### PR TITLE
react validated input

### PR DESCRIPTION
**Pull request by louiewhitz**
https://user-images.githubusercontent.com/99847558/198748384-ee65868e-af06-45af-9b4e-ab4041468827.mov

### CODE REVIEW

File: react-validated-input-component/src/validated-input.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {
+      password: '',
+      isValid: false,
+      errorMessage: 'Please enter a password'
+    };
+
+    this.tooLittleChar = this.tooLittleChar.bind(this);
+  }
```
Comment (TimelikeClosure) [inline react-validated-input-component/src/validated-input.jsx]: It doesn't look like you're using `isValid` or `errorMessage` in state, so I'd suggest removing them.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Comment by louiewhitz** [comment]
https://user-images.githubusercontent.com/99847558/198754741-1ab1688b-f06f-41a9-8311-bec180776ddb.mov

File: react-validated-input-component/src/validated-input.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+  tooLittleChar(event) {
+    this.setState({ password: event.target.value });
+    const password = event.target.value;
+    if (password.length < 7) {
+      this.setState({
+        password: event.target.value
+      });
+    }
```
Comment (scbowler) [inline react-validated-input-component/src/validated-input.jsx]: This code doesn't seem to really be doing anything. Here you only need the `setState` call to update `password` in state, nothing else is needed in this method.

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, just one update needed. Let me know if you have any questions.

**Comment by louiewhitz** [comment]
https://user-images.githubusercontent.com/99847558/199075355-e7846696-45e9-4ee2-8606-a01716aee678.mov

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**